src/components/errors/ErrorBoundaryInline.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+import Image from 'next/image';
+import { Component } from 'react';
+
+import { links } from 'src/config/links';
+import ErrorIcon from 'src/images/icons/error-circle.svg';
+import { logger } from 'src/utils/logger';
+
+interface ErrorBoundaryState {
+  error: any;
+  errorInfo: any;
+}
+
+export class ErrorBoundaryInline extends Component<any, ErrorBoundaryState> {
```

Comment:
a class ? haven't seen this in forever. if it works it works I suppose. Although I'm surprised you're not using [static getDerivedStateFromError](https://react.dev/reference/react/Component#static-getderivedstatefromerror)

Reply:
ah i just copied the existing error boundary and modified it. so i wasnt really thinking

Reply:
updated